app/styles/messaging-styles.css

```diff
@@ -4,14 +4,14 @@
 }
 
 .conversation-height {
-  height: calc(100vh - 225px);
+  height: calc(100vh - 175px);
```

Comment:
I wonder if we could refer to the height of the form directly here for maintenance sake 🤔 

I've not done this in CSS before, but presumably we can just have the `calc()` fetch a variable - maybe we can set it so that if the height of the form changes in future, developers don't need to find and manually change this value too, but the calculation considers the new value of it's height